app/helpers/analytics_helper.rb

```diff
@@ -25,4 +25,14 @@ def link_to_comment(comment, length)
   def link_to_comment_user(comment)
     link_to User.find(comment.user_id).name, user_path(User.find(comment.user_id))
   end
+
+  def case_updates
+    # returns all versions of all cases created in the last 30 days
+    PaperTrail::Version.where('created_at > ?', 30.days.ago)
```

Comment:
Why are you doing database queries in helpers?  Those should be in the controller.

Reply:
I can move them.

Reply:
Also, that query is going to blow up as the number of cases & versions gets larger. Is there a way to narrow the scope of this query?